src/utils/projection-mode.ts

```diff
@@ -1,4 +1,8 @@
 export enum ProjectionMode {
+  FLAT = 0,
+  SPHERE_EQUIRECTANGULAR = 1
+}
+export const enum PROJECTION_MODE {
```

Comment:
Maybe having `ProjectionMode` and `PROJECTION_MODE` enums may be confusing because the names are similar? How about renaming the latter to `ProjectionModeName` or something like that?